src/amber/cli/commands/new.cr

```diff
@@ -22,24 +23,29 @@ module Amber::CLI
 
       def run
         CLI.toggle_colors(options.no_color?)
-        full_path_name = File.join(Dir.current, args.name)
+        if args.name == "."
+          name = Dir.current.split("/").last
```

Comment:
I wasn't expecting to see manually splitting the name here and using basename below. Is there a reason for the difference?

Reply:
This allows you to bootstrap an existing project using `amber new .` I decided to grab the name of the project from the current directory.  Maybe there is a better way of getting the name?

Reply:
Sure, what I was wondering is why `File.basename Dir.current` doesn't work

Reply:
Aaah, gotcha.  I will fix that.